roles/reproducer/tasks/configure_controller.yml

```diff
@@ -21,7 +21,12 @@
   delegate_to: controller-0
   delegate_facts: false
   vars:
+    cifmw_sushy_emulator_hypervisor_target: "{{ inventory_hostname }}"
+    cifmw_sushy_emulator_install_type: podman
+    cifmw_sushy_emulator_hypervisor_target_connection_ip: "{{ inventory_hostname }}.utility"
```

Comment:
Now we are adding dns in to the mix this should be changed `connection_ip` to something like `connection_endpoint` or `connection_address`

Reply:
Renamed - might need to update the doc, but I'd rather ensure things are OK as-is.